### PR TITLE
add modeluri for model zoo to save

### PR DESCRIPTION
**Pull request by typhoonzero**
Part of https://github.com/sql-machine-learning/sqlflow/issues/1063

### CODE REVIEW

File: pkg/sqlfs/writer.go [hunk 1]
```diff
@@ -23,7 +23,8 @@ import (
 	pb "sqlflow.org/sqlflow/pkg/proto"
 )
 
-const bufSize = 4 * 1024
+// TEXT/STRING field support 64KB maximumn storage size
+const bufSize = 32 * 1024
```
Comment (shendiaomo) [inline pkg/sqlfs/writer.go:27]: Can we make this `bufSize` configurable? If my memory serves me right, the `string` type in MaxCompute has a much bigger size than 64KB.

File: pkg/sql/model.go [hunk 2]
```diff
@@ -19,22 +19,69 @@ import (
...
+				dir, file := path.Split(uriParts[1])
+				return m.saveTar(dir, file)
+			} else if uriParts[0] == "oss" {
+				return fmt.Errorf("save model to oss is not supported now")
+			}
+		} else {
+			return fmt.Errorf("error modelURI format: %s", modelURI)
+		}
+	}
+	db, err := NewDB(session.DbConnStr)
```
Comment (shendiaomo) [inline pkg/sql/model.go:53]: Is it possible to unify `sqlfs` with `modelURI`? For example, can we make `sqlfs://database_name/table_name` behave the same as `table_name`? This seems to improve consistency.

File: pkg/sqlfs/writer.go (hunk 1 above)
Comment (typhoonzero) [reply pkg/sqlfs/writer.go:27]: Better to keep it constant for better saved model migrations.

File: pkg/sql/model.go (hunk 2 above)
Comment (typhoonzero) [reply pkg/sql/model.go:53]: `oss://` and `file://` are well know URI prefixes. We do not intend to invent a new one.

**Review by shendiaomo — APPROVED** [review]

**Pull request merged**